pkgs/top-level/all-packages.nix

```diff
@@ -1610,7 +1610,9 @@ with pkgs;
 
   ddrescue = callPackage ../tools/system/ddrescue { };
 
-  deluge = python2Packages.deluge; # Package should be moved out of python-packages.nix
+  deluge = callPackage ../applications/networking/p2p/deluge {
+    pythonPackages = python2Packages;
```

Comment:
I'm asking mainly what we should do with these packages:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/all-packages.nix#L1223
https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/all-packages.nix#L6667
https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/all-packages.nix#L13524